controller_manager:
  ros__parameters:
    
    update_rate: 50

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster #Publishes /joint_states

    front_wheel_steer_position_controller:
      type: position_controllers/JointPositionController
      joints:
        - front_steer_joint
      ros__parameters:
        interface_name: position

    left_wheel_velocity_controller:
      type: velocity_controllers/JointVelocityController
      joints:
        - left_wheel_joint
      ros__parameters:
        interface_name: velocity

    right_wheel_velocity_controller:
      type: velocity_controllers/JointVelocityController
      joints:
        - right_wheel_joint
      ros__parameters:
        interface_name: velocity